src/Core/src/Handlers/View/ViewHandlerOfT.iOS.cs

```diff
@@ -44,7 +58,13 @@ public override Size GetDesiredSize(double widthConstraint, double heightConstra
 
 		protected override void SetupContainer()
 		{
+			var oldParent = NativeView?.Superview;
+			ContainerView ??= new WrapperView();
+
+			if (oldParent == ContainerView)
+				return;
 
+			ContainerView.MainView = NativeView;
 		}
 
 		protected override void RemoveContainer()
```

Comment:
What is supposed to happen when this runs? Remove things? Should the base handler be doing things? How does this affect layouts? For example a GridLayout references the view directly in its layout system, so changing the value of NativeView is going to cause issues.